tour/authorizations-code.md

```diff
@@ -1,64 +1,69 @@
 ---
 title: Authorizations Code
 ---
+
+Below is a solution for the exercise.
+
 ```java
-        // start writing your code here
-        // Connect to Mini Accumulo as the root user and create a table called "GothamPD".
-        Connector conn = mac.getConnector("root", "tourguide");
-        conn.tableOperations().create("GothamPD");
+static void exercise(MiniAccumuloCluster mac) throws Exception {
+    // Connect to Mini Accumulo as the root user and create a table called "GothamPD".
+    Connector conn = mac.getConnector("root", "tourguide");
+    conn.tableOperations().create("GothamPD");
 
-        // Create a "secretIdentity" authorization & visibility
-        final String secId = "secretIdentity";
-        Authorizations auths = new Authorizations(secId);
-        ColumnVisibility visibility = new ColumnVisibility(secId);
+    // Create a "viewSecretId" authorization & visibility
+    final String viewSecretId = "viewSecretId";
```

Comment:
I like "secretIdentity" better since it applies to both an authorization and visibility.  It seems odd to secure a column with a visibility = "viewSecretId"

Reply:
Fixed in ee62cbca.  I used `secretId` as it's shorted but I can go to `secretIdentity` if you want.

Reply:
That works.